experiments/scripts/delete_old_parquet.py

```diff
@@ -0,0 +1,58 @@
+from google.cloud import storage
+from pprint import pprint
+import sys
+from pathlib import Path
+sys.path.insert(0, f'{Path.cwd()}/')
```

Comment:
Is there a better way to handle this? In my experience, you can introduce a lot of annoying bugs when trying to do path inserts like this to force a script to run. Better to explicitly pip install your project `pip install -e <project-name>` so you can use it as an (editable) module, and don't have to worry about absolute vs relative import messes. Of course, to do so you'll need to get your `setup.py` working.

Reply:
The reason for this was that when I ran these scripts before putting this line in, it wouldn't recognize the `import config` command. This was one of the "strategies" that I was able to find on Stack Overflow. I can search/experiment to try and find another way to get it to work, or would you recommend I go with explicitly pip install my project?